.github/workflows/golangci-lint.yaml

```diff
@@ -24,7 +24,7 @@ jobs:
         uses: golangci/golangci-lint-action@v2
         with:
           # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
-          version: v1.43.0
+          version: v1.45.0
```

Comment:
v1.45.2 is the latest?

Reply:
I checked https://github.com/golangci/golangci-lint/releases and it seems like v1.45.2 is the latest. I will update the version and fix incoming problems, if any.